exploration/u-options.md

```diff
@@ -0,0 +1,149 @@
+# `u:` Options
+
+Status: **Accepted**
+
+<details>
+	<summary>Metadata</summary>
+	<dl>
+		<dt>Contributors</dt>
+		<dd>@eemeli</dd>
+		<dt>First proposed</dt>
+		<dd>2023-05-06</dd>
+		<dt>Pull Requests</dt>
+		<dd><a href="https://github.com/unicode-org/message-format-wg/pull/780">#780</a></dd>
+	</dl>
+</details>
+
+## Objective
+
+Provide common definitions for contextual options.
+
+## Background
+
+Function options may influence the resolution, selection, and formatting of annotated expressions.
+These provide a great solution for options like `minFractionDigits`, `dateStyle`,
+or other similar factors that influence the formatted result.
+
+However, this single bag of options is not appropriate in all cases,
+in particular for options that change the formatting context for the function.
```

Comment:
I'm not sure this is what you mean. Perhaps:

```suggestion
Such options naturally correspond to function arguments or builder-style function constructors.
Each option is specific to the associated API.
Users (message authors, translators, developers) should not expect
that a given option name has the same meaning between functions
or that its behavior stays the same.

To reduce the learning curve for users and improve consistency,
it would be useful to have common options 
(generally those related to the formatting context)
shared between all functions.
```